openwisp_monitoring/check/apps.py

```diff
@@ -1,8 +1,13 @@
 from django.apps import AppConfig
 from django.utils.translation import ugettext_lazy as _
 
+from .utils import manage_config_modified_retention_policy
+
 
 class CheckConfig(AppConfig):
     name = 'openwisp_monitoring.check'
     label = 'check'
     verbose_name = _('Network Monitoring Checks')
+
+    def ready(self):
+        manage_config_modified_retention_policy()
```

Comment:
how does this play with the abstraction? Can you make it play well with the changes introduced in #103?

Reply:
`manage_config_modified_retention_policy()` makes use of `timeseries_db.get_list_retention_policies` after rebasing which has been defined by us so it's abstract enough, I think.